packages/lib/src/components/Profile/components/ProfileDataCard/ProfileDataCard.tsx

```diff
@@ -92,6 +92,7 @@ export function ProfileDataCard({
             <PasswordChangeForm
               handlePasswordUpdate={handlePasswordUpdate}
               isLoading={isUpdatingPassword}
+              isAdmin={!!user.roles.find(role => role.name === 'ADMIN')}
```

Comment:
```suggestion
              isAdmin={Boolean(user.roles.find(role => role.name === ROLES.ADMIN)}
```

For consistency purposes 🙂 And you can import the enum `ROLES` from our types package. Would you mind changing this as well inside `RightsView.tsx` (app package) line `158`?

Reply:
Sure, good point 🎾